### DISCUSSION TITLE
Whisper2Summarize - Connecting Whisper and GPT to summarize your audio snippets!

### DISCUSSION THREAD

**Discussion by AndreDalwin** [openai/whisper#1208] [Show and tell]
Hi!

This is one of my first projects in Github as a CS student :))

I tried creating a python program with a GUI that transcribes Audio files then sends the transcript to GPT in order to summarize. (Since I'm using the GPT3.5 turbo API and not GPT4, I had to be creative in order to bypass the 4096 token limit in case the Audio file will exceed the token limit.)

Check it out here: [Whisper2Summarize](https://github.com/AndreDalwin/Whisper2Summarize)

LMK your thoughts on it or where I can improve, as I plan to further build on this as a hobby next time.

**Comment by ernestknurov** [comment]
Cool solution! I was doing similar task, you helped me lot with your code:)
